libs/gpu/src/dawn.zig

```diff
@@ -57,3 +59,19 @@ pub const TogglesDeviceDescriptor = extern struct {
         };
     }
 };
+
+const AdapterPropertiesPowerPreference = extern struct {
```

Comment:
Need to do the same thing described in https://github.com/hexops/mach/pull/672/files#r1070298339

dawn.json says:

https://github.com/hexops/dawn/blob/d8468ba09e23c482de1012e00b3adcf98dfe950f/dawn.json#L3013-L3021

Note that this is a chained `out` struct, and goes in `"chain roots": ["adapter properties"],` - so it corresponds to `gpu.Adapter.Properties.next_in_chain`